src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -257,7 +257,7 @@ class TutorialDirectoryUi extends React.Component {
           rightSideItems: headerLinks ? [headerLinks] : [],
         }}
       >
-        {this.renderTabContent()}
+        <KibanaPageTemplate.Section>{this.renderTabContent()}</KibanaPageTemplate.Section>
```

Comment:
I believe there is a regression with the content height. See how the white background isn't stretched to the bottom 

![Screenshot 2023-09-14 at 11 23 49](https://github.com/elastic/kibana/assets/7784120/a4db00a8-5575-4684-9f88-0a72c078cf21)

Reply:
Maybe it was like this before, but because the bg wasn't white it, it wasn't so noticeable

Reply:
This _might_ work, give it a shot:
```suggestion
        <KibanaPageTemplate.Section grow>{this.renderTabContent()}</KibanaPageTemplate.Section>
```

Reply:
The `[grow]` prop did not help, but if I remove `minHeight` in the inner template it grows to whole page size:

<img width="1618" alt="image" src="https://github.com/elastic/kibana/assets/82822460/2e304e97-e56b-42fe-9fa8-396d287a91b5">

Reply:
I guess I can remove this prop? Not sure why Kibana default would want to diverge from EUI default.

<img width="1222" alt="image" src="https://github.com/elastic/kibana/assets/82822460/63389d95-3dec-4d73-8294-105da879c713">

Reply:
If I remove it, there is extra scrollbar throughout Kibana, where there is actually nothing to scroll.

Reply:
@vadimkibana Try this:

```
diff --git a/packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx b/packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx
index 26829962406..05e9449903e 100644
--- a/packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx
+++ b/packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx
@@ -31,7 +31,6 @@ export const KibanaPageTemplateInner: FC<Props> = ({
   isEmptyState,
   pageSideBar,
   pageSideBarProps,
-  minHeight = 0,
   ...rest
 }) => {
   let header;
@@ -69,7 +68,8 @@ export const KibanaPageTemplateInner: FC<Props> = ({
       // the following props can be removed to allow the template to auto-handle
       // the fixed header and banner heights.
       offset={0}
-      minHeight={minHeight}
+      minHeight="calc(100vh - var(--euiFixedHeadersOffset, 0))"
+      grow={false}
       {...rest}
     >
       {sideBar}
```

(note: you'll have to unset the `"460px"` added to `tutorial_directory.js`)

Reply:
Thanks, I've added something similar to what you suggested: I'm using the `minHeight` and `grow` values that you suggested above, but if there is no `header` provided, I'm falling back the old values (before this PR: `minHeight:0` and `grow:undefined`).

This change could potentially affect existing pages that set header properties, but I did a look through Kibana pages, they seem to all look good.

Reply:
Should be resolved now.